x-pack/legacy/plugins/apm/server/routes/transaction_groups.ts

```diff
@@ -146,4 +147,23 @@ export function initTransactionGroupsApi(core: InternalCoreSetup) {
       }).catch(defaultErrorHandler);
     }
   });
+
+  server.route({
```

Comment:
This would need to change as well after you pull in the route changes from `master`.